src/client/components/Atoms/RectangleButton/RectangleButton.jsx

```diff
@@ -0,0 +1,22 @@
+import styles from './RectangleButton.module.css';
+
+export const RectangleButton = ({ title, type = 'button', size, onClick = () => {} }) => {
```

Comment:
Exports: sticking to the estabilished convention declare component as it is without directly exporting it.
Instead use last line to export button

f.ex. 
`export { RectangleButton }`

just in case if we will export more than one variable from file

Reply:
It doesn't stops us from exporting more variables. Just in each new you add export const or function or whatever. And import is the same syntax. For example
` import { function1, variable1, ...etc} from 'pathToFile.js'`